grapher/file_grapher.py

```diff
@@ -303,9 +303,15 @@ def _get_module_parent_from_module_path(module_path):
         return parent_module.replace(os.sep, '.')
 
     def _abs_module_path_to_relative_module_path(self, module_path):
-        rel_path = os.path.relpath(module_path, self._base_dir)
-        if not rel_path.startswith('..'):
-            return rel_path
+        rel_path = module_path
+        try:    
+            rel_path = os.path.relpath(module_path, self._base_dir)
+            if not rel_path.startswith('..'):
+                return rel_path
+        except ValueError:
+            # (alexsaveliev) virtualenv may use python from the different location.
```

Comment:
should an exception be thrown in this case or an error printed to stderr, instead of silently failing?